### PR TITLE
OLS-671: /readiness - LLM check

### PR DESCRIPTION
**Pull request by onmete**
## Description

Add LLM check for /readiness probe.

## Type of change

- [x] New feature

### CODE REVIEW

**Comment by bparees** [comment]
/approve

looks like some test fixes are needed

File: ols/app/endpoints/health.py [hunk 1]
```diff
@@ -5,12 +5,44 @@
...
+    global llm_is_ready_persistent_state
+    if llm_is_ready_persistent_state is True:
+        return True
+    try:
+        bare_llm = load_llm(
+            config.ols_config.default_provider, config.ols_config.default_model
+        )
+        response = bare_llm.invoke(input="Hello there!")
+        # BAM and Watsonx replies as str and not as `AIMessage`
+        if isinstance(response, str) or isinstance(response, AIMessage):
```
Comment (tisnik) [inline ols/app/endpoints/health.py:37]: can be written as `isinstance(response, [str, AIMessage])`

**Review by tisnik — COMMENTED** [review]
it seems ok, just one nitpick

**Pull request merged**